tests/kernel/install_ltp.pm

```diff
@@ -96,6 +96,14 @@ sub run {
     my $self     = shift;
     my $inst_ltp = get_var 'INSTALL_LTP';
     $self->wait_boot;
+
+    # poo#18980
+    # Always add serial console (even if virtio console is not used) as tests
+    # using run_ltp.pm might be using it.
+    if (check_var('ARCH', 'ppc64le')) {
+        add_serial_console('hvc1');
```

Comment:
We can just remove this if it is in shutdown.pm. Also it won't work without first selecting root console.